core/proto_test.go

```diff
@@ -64,6 +66,18 @@ func TestParSignedDataSetProto(t *testing.T) {
 				},
 			},
 		},
+		{
+			Type: core.DutyBuilderProposer,
+			Data: core.VersionedSignedBlindedBeaconBlock{
+				VersionedSignedBlindedBeaconBlock: eth2api.VersionedSignedBlindedBeaconBlock{
+					Version: spec.DataVersionBellatrix,
+					Bellatrix: &eth2v1.SignedBlindedBeaconBlock{
+						Message:   testutil.RandomBellatrixBlindedBeaconBlock(t),
+						Signature: testutil.RandomEth2Signature(),
+					},
```

Comment:
can extract this to `testutil.RandomCoreVersionSignedBlindedBeaconBlock` is you want to decrease creating these from scratch every time.